pokeapp/src/components/SidebarNav.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+
+export const SidebarNav = ({ setIsBadgeSideBar, setbadgeBtnActive1, setbadgeBtnActive2, badgeBtnActive1, badgeBtnActive2 }) => {
+  return (
+    <>
+      <div
+        className={badgeBtnActive1 ? "badge-btn-left active" : "badge-btn-left"}
+        onClick={() => {
+          setbadgeBtnActive1(true);
+          setbadgeBtnActive2(false);
+          setIsBadgeSideBar(true);
+        }}
+      >
+        Badges
+      </div>
+      <div
+        className={
+          badgeBtnActive2 ? "badge-btn-right active" : "badge-btn-right"
```

Comment:
nice so what I am getting is they come in and out depending on the state the user chooses